src/views/Categories.vue

```diff
@@ -42,20 +67,57 @@ export default {
         this.categories = defaultOptions;
       }
     },
+    getQuestions() {
+      const defaultQuestions = {
+      };
+
+      this.categories.forEach((category) => {
+        defaultQuestions[category.value] = [];
+      });
+      console.log(defaultQuestions);
+
+      if (localStorage.getItem('questions')) {
+        this.questions = JSON.parse(localStorage.getItem('questions'));
+      } else {
+        this.questions = defaultQuestions;
+      }
+    },
     addCategory() {
       this.categories.push({
         text: this.newCategoryName,
         value: this.newCategoryName.trim().toLowerCase(),
       });
-
+      this.questions[this.newCategoryName.trim().toLowerCase()] = [];
+      localStorage.setItem('questions', JSON.stringify(this.questions));
       localStorage.setItem('categories', JSON.stringify(this.categories));
       this.newCategoryName = '';
     },
+    deleteCategoryControl() {
+      console.log(this.newCategoryName);
+      console.log(this.categories.length);
+      let i = 0;
+      this.questionsControl = JSON.parse(localStorage.getItem('questions'));
+      while (this.categories.length > i) {
```

Comment:
this part can probably be simpler if you avoid i and define and use variables for frequently used values like `const aGoodNameForYourConstant = this.newCategoryName.trim().toLowerCase();`